Equinox/Level.cpp

```diff
@@ -9,10 +9,21 @@
 #include "ModuleWindow.h"
 #include "IMGUI/imgui.h"
 #include "ModuleEditor.h"
+#include "Quadtree.h"
+#include "ModuleEditorCamera.h"
 
 Level::Level()
 {
 	root = new GameObject;
+
+	vec minPoint = vec(-10000, -100, -10000);
+	vec maxPoint = vec(10000, 100, 10000);
+
+	AABB limits = AABB(minPoint, maxPoint);
+
+	quadtree = new Quadtree(limits);
+
+	//quadtree->Insert(root);
```

Comment:
it's because we are thinking if it is or not necessary to add root to the quadtree, finally we can consider that is redundant to add it... because it is just a big AABB that encloses all the scene.